lib/generators/manageiq/plugin/templates/lib/%plugin_path%/engine.rb

```diff
@@ -11,5 +11,13 @@ def self.vmdb_plugin?
     def self.plugin_name
       _('<%= plugin_human_name %>')
     end
+
+    def self.init_loggers
+      $<%= plugin_name %>_log ||= Vmdb::Loggers.create_logger("<%= plugin_name %>.log")
+    end
+
+    def self.apply_logger_config(config)
+      Vmdb::Loggers.apply_config_value(config, $<%= plugin_name %>_log, :level_<%= plugin_name %>)
```

Comment:
If you go with my suggestion above for commenting the contents of `init_loggers` out, you would probably want to do that here as well.